docs/policies.md

```diff
@@ -180,7 +199,7 @@ spec:
   - Ingress
 ```
 
-RKE2 applies 2 network policy for all namespaces and updates the namespaces with annotations `default-network-policy` and `default-network-dns-policy`. 
+RKE2 applies 2 network policy for all built-in namespaces and updates the namespaces with annotations `np.rke2.io` and `np.rke2.io/dns`.
```

Comment:
All built-in namespaces get the one policy but only kube-system gets the dns policy.

Reply:
So should this read
```
RKE2 applies the `default-network-policy` policy and `np.rke2.io` annotation to all built-in namespaces. The `kube-system` namespace additionally gets the `default-network-dns-policy` policy and `np.rke2.io/dns` annotation applied to it.
```
?

Reply:
Yeah, that works.